Update README.md (#11)
mesg-foundation/handbook: token/README.md

[[toc]]

## Algorithmic Token Distribution

## Exchange Listing
The activities below represent a checklist of involved in getting MESG listed on Token Exchanges.

> While a linear process my be implied below, it is accepted that each Exchange interaction won't always follow this exact path. However, this checklist should still be adhered to as much as possible.

Reviewer: ```suggestion
While a linear process may be implied below, it is accepted that each Exchange interaction won't always follow this exact path. However, this checklist should still be adhered to as much as possible.
```

> Throughout the life-cycle of the Exchnage Lisring 

Reviewer: ```suggestion
Throughout the life-cycle of the Exchange Listing 
```

1. Select an Exchange

> * YELLOW will select the ideal Exchange to approach from the pool of pre-approved echanges in the MESG Token Exchanges Google Doc located here: https://docs.google.com/spreadsheets/d/1Ks4YaVGldGDJd73k9IcMx2EDUfKJiKZlHcLBfPDe3mA/edit#gid=0

Reviewer: ```suggestion
* YELLOW will select the ideal Exchange to approach from the pool of pre-approved exchanges in the MESG Token Exchanges Google Doc located here: [https://docs.google.com/spreadsheets/d/1Ks4YaVGldGDJd73k9IcMx2EDUfKJiKZlHcLBfPDe3mA/edit#gid=0](https://docs.google.com/spreadsheets/d/1Ks4YaVGldGDJd73k9IcMx2EDUfKJiKZlHcLBfPDe3mA/edit#gid=0)
```

* The engagement priority is reflected in listing order.

2. Engage with Exchange

YELLOW will initiate contact and start discussions with the selected Exchange. The purpose of the engagement is to obtain information about listing procedure such as: 

* Listing Price
* Timeframe
* What marketing services they can offer
* Other information that will affect the listing process.

It is imperative that the above information is communicated back to MESG throughout the engagement.

3. MESG and Yellow reach consensus on whether the listing is to be done.

4. MESG completes the application process.

5. YELLOW assess and negotiates the Exchange contract. 

> Details of this activity must always communicated to MESG. For instance: which aspects of the contract are problematic, beneficial and what the status of the negotiation is. 

Reviewer: ```suggestion
Details of this activity must always be communicated to MESG. For instance: which aspects of the contract are problematic, beneficial and what the status of the negotiation is. 
```

6. MESG select listing date.
Yellow's input on this topic will be sought, but the criteria for selecting the listing date will typically involve providing enough time to create marketing material and visibility.

> 7. MESG to set up marketing effort in conjunction with the Exhange's offerings and Yellow's guidance. It is likely that the marketing campaigns for all exchange listings will look mostly alike and that Standard Operating Procedures can be designed for these. But, given that some exchanges will provide additional marketing support and will have unique offerings, each listing's marketing campaign will also have a bespoke component.

Reviewer: ```suggestion
7. MESG to set up marketing effort in conjunction with the Exchange's offerings and Yellow's guidance. It is likely that the marketing campaigns for all exchange listings will look mostly alike and that Standard Operating Procedures can be designed for these. But, given that some exchanges will provide additional marketing support and will have unique offerings, each listing's marketing campaign will also have a bespoke component.
```
